src/Kunstmaan/AdminListBundle/Controller/AdminListController.php

```diff
@@ -105,7 +105,6 @@ protected function doAddAction(AbstractAdminListConfigurator $configurator, $typ
 
         /* @var EntityManager $em */
         $em = $this->getEntityManager();
-        $repo = $em->getRepository($configurator->getRepositoryName());
```

Comment:
Setting the $repo variable was moved to the new method, as it was not used anywhere else.